### ISSUE TITLE
refining tagging - advance help request

### ISSUE THREAD

**Issue by jkoul** [ga-wdi-exercises/project2#134]
So I installed this gem called "acts-as-taggable-on" in order to create a single tagging framework applicable to both users and posts. I managed to set it up for users, but only after incorporate a controller and model for tags, which I wasn't supposed to have to do per gem documentation. Then, when I tried to work in posts, it messed things up so badly that I ended up having to revert to the last commit.

The gem is at https://github.com/mbleigh/acts-as-taggable-on, if that helps.

I'll be in at 9 tomorrow so hopefully can start troubleshooting this challenge right away. Your feedback and assistance are appreciated.

**Comment by RobertAKARobin** [comment]
Can you provide us with some specific errors and code to which we can respond, Jeremy?

**Comment by jkoul** [comment]
Well I think it might be an issue with how I installed the gem from the git-go, because all the documentation says I didn't need a model for tags and it kept asking for a model for tags. My hunch is that my best bet is to get rid of the gem entirely and start tagging functionality again from scratch (either with the gem or not, TBD). I'd use a "rails destroy acts-as-taggable-on" to roll it back, right?

**Comment by RobertAKARobin** [comment]
First, `rake db:rollback`. This will remove the previous migration, which presumably is the one in which you added the tags. Then remove acts-as-taggable from your Gemfile, and `bundle install` to remove it from your app. Then recreate your database.

From this point, you can either try installing acts-as-taggable again, making sure you go step-by-step through the documentation, or you can make the Tag model yourself and do it that way. If you go the Gem route, unless you provide specific errors we'll have to put that at a lower priority to other issues since it's outside the scope of what we've covered in class.

**Comment by jkoul** [comment]
I'm going for manually. I actually had it partially implemented already before trying out the gem. Thanks for the guidance. No worries about low priority on the issue - it's not your job to help me troubleshoot custom gem implementation. And I like hardcoding stuff anyway at this stage of my development.
